.pre-commit-config.yaml

```diff
@@ -1,124 +1,87 @@
 repos:
-  # Prevent commits to the main branch.
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.4.0
+    rev: v4.6.0
     hooks:
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-json
+      - id: check-merge-conflict
+      - id: check-toml
+      - id: check-yaml
+      - id: debug-statements
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+        args:
+          - --fix=auto
+      - id: name-tests-test
+        args:
+          - --pytest-test-first
       - id: no-commit-to-branch
-        name: ensure we are not committing to branch `main`
         args:
           - --branch=main
-
-  # Make sure all imports are absolute in Python files.
-  - repo: https://github.com/MarcoGorelli/absolufy-imports
-    rev: v0.3.1
-    hooks:
-      - id: absolufy-imports
-        name: format Python imports using absolufy-imports
+      - id: trailing-whitespace
 
   # Sort Python imports using `isort`.
   - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+    rev: 5.13.2
     hooks:
       - id: isort
-        name: sort Python imports using isort
         args:
           - --force-grid-wrap=0
-          - --line-length=100
+          - --line-length=88
```

Comment:
nitpick, I think we use 100 in many arviz projects (and Pymc ones too)

Reply:
Thanks. I am going to update the pre-commit to use `ruff`, since that is what is used in other repos. I updated the `pyproject.toml` file without even looking at the pre-commit one.